### PR TITLE
How to install IDE plug-ins

### PR DESCRIPTION
**Pull request by lordqwerty**
Notes added for how to install each of the plug-ins available for editing spthy files.

### CODE REVIEW

File: src/002_installation.md [hunk 1]
```diff
@@ -213,3 +213,45 @@ running even if the network drops your connection as you can later
...
+--------------------
+
+Under the [etc](https://github.com/tamarin-prover/tamarin-prover/tree/develop/etc) folder contained in the Tamarin Prover project, plug-ins are available for VIM, Sublime Text 3 and Notepad++. Below we details the steps required to install your preferred plug-in.
+
+### VIM ###
+
+1. Create `.vim` directory in your home directory if not already existing
+2. Change directory to `.vim`
+3. Place the [filetype.vim]() file
+4. Create another directory named `syntaxes` in the `.vim` directory and change to it
```
Comment (cascremers) [inline src/002_installation.md]: Shouldn't this be 'syntax'? (or technically, $VIMRUNTIME/syntax)

Comment (lordqwerty) [inline src/002_installation.md]: Don't really use VIM too heavily. This is how I have it setup on my machine but perhaps this is the preferred method?

I was following the [Install the syntax file section from the VIM wiki](http://vim.wikia.com/wiki/Creating_your_own_syntax_files).

Comment (cascremers) [inline src/002_installation.md]: To the best of my knowledge, `$VIMRUNTIME/syntaxes`, even if it works, is not a documented feature. The VIM help states `$VIMRUNTIME/syntax` (so typically `~/.vim/syntax/` ). Also the VIM wiki that you point to uses `/syntax/` and not `/syntaxes/`. So I suggest changing it to `syntax`.

**Comment by rsasse** [comment]
I take it this is now complete and ready to be included in the manual? It should be included in conjunction with including @202 of tamarin-prover, right? That also seems ready.

**Comment by lordqwerty** [comment]
This is now complete and I believe ready to be included.

**Comment by rsasse** [comment]
Excellent, thank you for your effort!

**Pull request merged**